src/Dom/scrollLocker.ts

```diff
@@ -17,6 +18,31 @@ const scrollingEffectClassNameReg = new RegExp(
   'g',
 );
 
+let scrollPosition = 0;
+
+const setContainerStyle = (
+  container: HTMLElement,
+  scrollBarSize: number,
+): React.CSSProperties => {
+  const defaultStyle: React.CSSProperties = {
+    width: `calc(100% - ${scrollBarSize}px)`,
+    overflow: 'hidden',
+    overflowX: 'hidden',
+    overflowY: 'hidden',
+  };
+
+  if (container === document.body) {
```

Comment:
Can we judge mobile?

Reply:
@shaodahong , what do you mean? I don't understand, in a mobile view, body is also present in the DOM

Reply:
![incorrect-ipad-body](https://user-images.githubusercontent.com/28491189/117995268-dd32a380-b349-11eb-910b-8ce4d8df385e.gif)

Reply:
@shaodahong , I checked on IPad with isMobile() flag and work incorrect

Reply:
on Tablet devices isMobile() -> return false

Reply:
>  I checked on IPad with isMobile() flag and work incorrect

Yep, so we need judge like this 

```js
const isIosDevice = (/iP(ad|hone|od)/.test(window.navigator.platform)

if (isIosDevice) {
//...
}

```

Reply:
@shaodahong, on IPad devices now platform return MacIntel
![image](https://user-images.githubusercontent.com/28491189/118285241-ed798880-b4d9-11eb-8eea-e3738d88fc02.png)